test/e2e/aws_test.go

```diff
@@ -132,13 +139,113 @@ var _ = Describe("functional tests", func() {
 			By("Scale the MachineDeployment up again")
 			scaleMachineDeployment(namespace, machineDeploymentName, scaleDownReplicas, initialReplicas)
 
+			By("Creating the LB service")
+			sess := getSession()
+			elbClient := elb.New(sess)
+			createLBService(namespace, clusterName, lbServiceName)
+			// This sleep is required to let the LB service up and running
+			time.Sleep(15 * time.Second)
+			verifyElbExists(elbClient, lbServiceName, true)
+
 			By("Deleting the Cluster")
 			deleteCluster(namespace, clusterName)
 
+			By("Checking provisioned LB deleted")
+			verifyElbExists(elbClient, lbServiceName, false)
 		})
 	})
 })
 
+func createLBService(namespace, clusterName, svcName string) {
```

Comment:
Suggestion: maybe at a later stage it will useful to create a generic `createService` function which takes a `Service.Spec`

Reply:
Even though creating the service Spec is the major code addition than creating the service, added a generic method which accepts spec and labels